package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "Globalization Pipeline Extension",
     "description": "Connect to the Globalization Pipeline service on Bluemix",
     "icon": "images/logo.svg",
-    "version": "0.1.0",
+    "version": "0.1.1",
```

Comment:
I think when we publish this, it will see version in package.json and then bump it